lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.html

```diff
@@ -26,7 +26,7 @@ <h4 class="adf-aspect-list-dialog-title">{{title | translate}}</h4>
         </button>
     </div>
     <div>
-        <button mat-button cdkFocusInitial (click)="onCancel()" id="aspect-list-dialog-actions-cancel"
+        <button mat-button adf-auto-focus (click)="onCancel()" id="aspect-list-dialog-actions-cancel"
```

Comment:
Is that still a bug? I just tried open that screen on remove environment and Cancel button is focused.

Reply:
@AleksanderSklorz if you will open a file and choose from the top menu (3 dots -> Edit Aspects) you will notice that the focus remains on the menu, but not on the Cancel btn.

Reply:
ah ok I see, thanks :)